ui.tests/test-module/specs/panelcontainer/panelcontainer.runtime.complex.spec.js

```diff
@@ -193,7 +193,9 @@ describe( "Form Runtime with Panel Container complex repeatability use cases ",
         const innerModelId = instancesModel.id;
         const numberInputId = instancesModel[index].items[0].id;
         const checkBoxGroupId = instancesModel[index].items[1].id;
-        cy.get(`#${innerModelId}`).should('exist');
+        return cy.get(`#${innerModelId}`).should('exist');
+        //TODO: change to bem based selectors by using get view APIs.
+        /*
         cy.get(`#${innerModelId}-label`).should('exist');
```

Comment:
we should fix this instead of commenting. I commented it in my PR as I didn't have the bandwidth to take this up.